lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -291,6 +295,20 @@ def supported_features(cls) -> List[Type[Feature]]:
             features.StartStop,
         ]
 
+    def _get_image_info(
```

Comment:
move to the end, the _prepapre_environment and several others are more important, should be on the top.